src/test/java/bio/terra/integration/DataRepoFixtures.java

```diff
@@ -367,13 +368,26 @@ private SnapshotSummaryModel finishCreateSnapshot(
         return snapshotResponse.getResponseObject().get();
     }
 
-    public DataRepoResponse<SnapshotModel> getSnapshotRaw(TestConfiguration.User user, UUID snapshotId)
+    public DataRepoResponse<SnapshotModel> getSnapshotRaw(TestConfiguration.User user,
+                                                          UUID snapshotId,
+                                                          List<SnapshotRequestAccessIncludeModel> include)
         throws Exception {
-        return dataRepoClient.get(user, "/api/repository/v1/snapshots/" + snapshotId, SnapshotModel.class);
+        String includeParam;
+        if  (include != null && !include.isEmpty()) {
```

Comment:
whitespace; also pretty sure you don't need `isEmpty()` here; if there's no elements then you get `?` which should be OK.
```suggestion
        if (include != null && !include.isEmpty()) {
```

Reply:
Also, why allow `null` at all? This method should require an empty list if a caller doesn't want to pass in the include params.

Reply:
null is a valid input to the endpoint and so I wanted to support that too

Reply:
So a user can provide `null` in the URL and prevent the api default from being used? That doesn't seem right to me, have you verified that it's possible for `null` to reach here?